src/main/kotlin/no/nav/helse/prosessering/v1/PreprossesertMeldingV1.kt

```diff
@@ -65,12 +68,15 @@ data class PreprossesertSoker(
 
 data class PreprossesertBarn(
     val fodselsnummer: String?,
-    val navn : String?,
+    val navn: String?,
     val alternativId: String?,
     val aktoerId: String?
 ) {
-    internal constructor(barn: Barn, barnetsNavn: String?, aktoerId: AktoerId?) : this(
-        fodselsnummer = barn.fodselsnummer,
+
+    internal constructor(
+        barn: Barn, barnetsNavn: String?, barnetsFodselsnummer: NorskIdent?, aktoerId: AktoerId?
```

Comment:
`NorskIdent?` paramteret her og & inn i `PreprossesertMeldingV1` burde hete noe sånt som `barnetsNorskeIdent` ettersom det ikke nødvendigvis er et fødselsnummer.

Videre logikk i `PreprosessertBarn` sin constructor blir da noe sånt som dette;

`
    internal constructor(
        barn: Barn, barnetsNavn: String?, barnetsNorskeIdent: NorskIdent?, aktoerId: AktoerId?
    ) : this(
        fodselsnummer = if (barnetsNorskIdent is Fodselsnummer) barnetsNorskeIdent.getValue() else barn.fodselsnummer,
        navn = barnetsNavn,
        alternativId = if (barnetsNorskIdent is AlternativId) barnetsNorskeIdent.getValue() else barn.alternativId,
        aktoerId = aktoerId?.id
    )
`

Reply:
Glemte helt alternativId. Nice catch 👍